src/containers/transfer/components/EthTransferForm.js

```diff
@@ -133,7 +133,7 @@ class EthTransferForm extends React.Component {
                             type="submit"
                             loading={submitting}
                             data-testid="submitEthTransferButton"
-                            className={submitting ? "loading" : ""}
```

Comment:
no more progress spinner. intentional? (to be clear: i don't particularly care)

Reply:
yep, cause these were removed from all the other forms.
instead, they use the following wording:
{submitting ? "Submitting..." : submitText || "Send"}

Reply:
btw the spinner was not spinning lately...